### PR TITLE
Branch ug2

### PR DESCRIPTION
**Pull request by a1021492980**
This PR contains the UG for search features and updated help features

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -513,32 +727,104 @@ Format: `help`
...
+	 up if exist.
+	 search /workout 15
+	 This will show the workout(s) with reps = 15 if exist.
+----------------------------------------------------------------------
+	 To search for plan(s), please enter: 
+	 search /plan <plan keyword>
+	 Example: 
+	 search /plan grow
+	 This will show the plan(s) containing keyword grow if exist.
+----------------------------------------------------------------------
```
Comment (Musfirahe0556596) [inline docs/UserGuide.md:827]: is search /all inside?

Comment (Musfirahe0556596) [inline docs/UserGuide.md:827]: is search /all inside?

**Review by Musfirahe0556596 — COMMENTED** [review]
Looks good! Just a question!

File: docs/UserGuide.md (hunk 1 above)
Comment (a1021492980) [reply docs/UserGuide.md:831]: Just added that, thank you for finding it!

**Pull request merged**